File: hotel/sample/resources/resources/packages/testpack/binder22.py
import util
import cutil

def dialogaction(action,var) :
  cutil.printVar("binder1",action,var)
  
  if action == "button1" :
    cutil.setBinderOpen(var,"toast1")

  if action == "button2" :
    cutil.setBinderOpen(var,"toast2")

File: hotel/sample/resources/resources/dialogs/js/binder22.js
var TOAST22 = (function() {
   var my = {};      
   
   my.opentoast1 = function (s) {
     CUTIL.debug(s)
     var res = {};
     CUTIL.setBinderOpen(res,"toast1");       
     return res;
   };
   my.opentoast2 = function (s) {
     CUTIL.debug(s)
     var res = {};
     CUTIL.setBinderOpen(res,"toast2");
     return res;
   };
   return my;
}());